app/client/src/components/pages/State/components/tabs/WaterQualityOverview.js

```diff
@@ -967,6 +972,7 @@ function WaterQualityOverview({ ...props }: Props) {
           }
         >
           <AccordionContent>
+            <em>Stories below open in a new browser tab.</em>
```

Comment:
GitHub won't let me add a comment to an unchanged line, so this comment doesn't have anything to do with this line.

In the drinking water section of the drinking water tab of the state water quality overview page, there are two links that open up new tabs. 
1. The "EPA’s Drinking Water Performance and Results (GPRA) Report" link is in the `WaterSystemSummary` component. This one I think could be covered by putting `Links on this page open in a new browser tab.` text above the usage of the `WaterSystemSummary` component. I think it's best here rather than in the component, because the national page also uses this component and already has the new tab disclaimer. 
2. On line 923, there is a link that opens an ofmpub url in a different tab. In the app this in the drinking water section of the state water quality overview page. I think this probably needs to be an EXIT disclaimer. If it doesn't need to be an exit disclaimer, then it would be covered by putting the new tab disclaimer above the usage of the `WaterSystemSummary` component. @coobr01 What are your thoughts on this one?

Reply:
@cschwinderg OFM is still internal to EPA so no exit disclaimer is needed, we should be covered by the normal popup text, thanks!